pilot/pkg/networking/util/util.go

```diff
@@ -195,6 +195,30 @@ func BuildAddress(bind string, port uint32) *core.Address {
 	}
 }
 
+// BuildAddress returns a SocketAddress with the given ip and port or uds.
+func BuildAddressV3(bind string, port uint32) *corev3.Address {
```

Comment:
Since this is going to be longer in the codebase, I suggest we call this `BuildAddress` and the other one `BuildV2Address`. We can remove `Buildv2Address` when listener and routes are also moved to v3. Alternatively we can move these to functions to v2Util or some thing like that and can delete that file later. Same comments for other functions that were suffixed with V3

Reply:
done